Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml

```diff
@@ -4050,7 +4050,78 @@ script:
       type: string
     description: Gets the public IP addresses that have been discovered by IPAM.
     name: aws-ec2-get-ipam-discovered-public-addresses
-  dockerimage: demisto/boto3py3:1.0.0.87088
+  - arguments:
+    - description: The ID of the VPC in which the endpoint will be used.
+      name: vpcId
+      required: true
+    - description: The service name for the service that you want to create an endpoint.
+      name: serviceName
+      required: true
+    - description: The type of endpoint.
+      name: endpointType
+    - description: One or more subnet IDs in which to create the endpoint.
+      name: subnetIds
+      isArray: true
+    - description: One or more security group IDs to associate with the endpoint.
+      name: securityGroupIds
+      isArray: true
+    - description: Checks whether you have the required permissions for the action, without actually making the request.
+      name: dryRun
+      isArray: false
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+    - description: The type of endpoint.
+      name: vpcEndpointType
+      isArray: false
+      auto: PREDEFINED
+      predefined:
+      - "Interface"
+      - "Gateway"
+      - "GatewayLoadBalancer"
+    - description: A policy document to attach to the endpoint. A JSON policy document that controls access to the service from the endpoint.
+      name: policyDocument
+    - description: One or more route table IDs.
+      name: routeTableIds
+      isArray: true
+    - description: Unique, case-sensitive identifier to ensure the idempotency of the request.
+      name: clientToken
+    - description: Indicates whether to associate a private hosted zone with the specified VPC.
+      name: private-dns-enabled
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+    - description: One or more tags to associate with the endpoint. Should be Json string of key-value tags.
+      name: tagSpecifications
+    - description: The AWS Region, if not specified the default region will be used.
+      name: region
+    - description: The Amazon Resource Name (ARN) of the role to assume.
+      name: roleArn
+    - description: An identifier for the assumed role session.
+      name: roleSessionName
+    - description: The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) up to the maximum session duration setting for the role.
+      name: roleSessionDuration
+    outputs:
+    - contextPath: AWS.EC2.Vpcs.VpcEndpoint.VpcEndpointId
```

Comment:
As discussed should be changed to: AWS.EC2.VPC.VPCEndpoint.VpcEndpointId (confirm with @altmannyarden )